### ISSUE TITLE
what happened？

### ISSUE THREAD

**Issue by KaWah1222** [roro08/Sketchfab-Ripper#23]
![image](https://user-images.githubusercontent.com/51141646/168776256-6055e74b-4c0c-42ea-828d-b498ae2142f9.png)

**Comment by moumouh206** [comment]
Open the sketchfab-ripper.ini file in same directory with any text editor and replace the key "sfConfig" with this new encryption key 👍 sfConfig=iqmKWuccsR1DWJLcPTT92OnLU1X94yCrmo5gswmXJ9Y7ADy3XoAfyXab4HFrkUs4yqxZb8ligH6eEMMZvVagj3UPESR0fhbNI4W6r/ezBVQuY86q5hsFUDpTc9UERZGZ

**Comment by musuvataf** [comment]
how can we get this encryption key when it changed?

**Comment by moumouh206** [comment]
sketchfab source if you know what's you're doing

**Comment by DLewis01** [comment]
@moumouh206 perhaps you might tell us what js file it is in so we can have a go at either decrypting it, or writing an extractor.

**Comment by moumouh206** [comment]
@DLewis01 go to this discord server, you'll find what you need 👍 [https://discord.gg/C8nnf5kvhv](url)

**Comment by TITANDERP** [comment]
> Open the sketchfab-ripper.ini file in same directory with any text editor and replace the key "sfConfig" with this new encryption key 👍 sfConfig=iqmKWuccsR1DWJLcPTT92OnLU1X94yCrmo5gswmXJ9Y7ADy3XoAfyXab4HFrkUs4yqxZb8ligH6eEMMZvVagj3UPESR0fhbNI4W6r/ezBVQuY86q5hsFUDpTc9UERZGZ

bless your soul

**Comment by LEV1TEK** [comment]
I dropped it in still same error, how often does it change or am i doing something wrong?

**Comment by KaWah1222** [comment]
yep on work emm...

**Comment by KaWah1222** [comment]
yep no work emm

**Comment by KaWah1222** [comment]
OK, it worked but there is a new problem

**Comment by moumouh206** [comment]
@KaWah1222 what's the new problem?
this is the new encryption key : sfConfig=sxZBsItQh5uyH9Whd+cus0bNsYmGTowIyKLB7LqzoES66mXVwRKTJagnVUwS9+gI5dLtcEjW7NN+tRccSZ6J1UftvLzJhq1bmfa1rQS72qzyzy30zvIvOfAeZMGS5NiO

**Comment by nbzottel** [comment]
I'm also getting the Newtonsoft.json error (like OP); the new encryption key does not help. Additionally, after editing the ini, trying to rip a model and failing, I checked the ini again: The encryption key had been changed to:

sfAccessToken=
sfAuthType=
**sfConfig=JLBSh92k6kkPHDWy8DAuyptA5UMoOIJs0agAyH8636Ik/Kf+IG5oAuCpnY+fW+XN**
sfLastAuth=0
sfRefreshToken=
sfSessionID=

Could you help me / us out again?

**Comment by moumouh206** [comment]
This is the new encryption key 👍 sfConfig=JLBSh92k6kkPHDWy8DAuyptA5UMoOIJs0agAyH8636LhuwOyBArA+EC3pOw9aEABmJi67JPbPd79jb3amTRXDYSBZ+cFZxPE9GZrOKPkpZMcaZlW3qc9Hd/ohrISBnJF

**Comment by nbzottel** [comment]
Wow! That actually did it. Thank you for the quick help.

**Comment by nbzottel** [comment]
> This is the new encryption key 👍 sfConfig=JLBSh92k6kkPHDWy8DAuyptA5UMoOIJs0agAyH8636LhuwOyBArA+EC3pOw9aEABmJi67JPbPd79jb3amTRXDYSBZ+cFZxPE9GZrOKPkpZMcaZlW3qc9Hd/ohrISBnJF

Because you were able to help out so quickly: Do you know if the Sketchfab shaders / textures also get decrypted? They look awefully messy - but maybe that is just the Blender noob talking. Do you have any experiences working with Blender / importing shaders and could point me in the right direction? :)

**Comment by zsna** [comment]
Why does it crash after entering a new encryption key and opening it after pasting the download link

**Comment by moumouh206** [comment]
@nbzottel the texture are encrypted and no actual fix for that , the only thing to do is to extract the textures manually using Firefox/chrome ( [https://spector.babylonjs.com/](https://github.com/roro08/Sketchfab-Ripper/issues/url) ) extension called spector.js
here is a quick tutorial : https://www.youtube.com/watch?v=klttU44JAfQ

**Comment by moumouh206** [comment]
> Why does it crash after entering a new encryption key and opening it after pasting the download link

Maybe you should run it as administrator

**Comment by moumouh206** [comment]
@nbzottel https://mega.nz/file/wQl0XISA#khEE6DQ2TM6OvvGExgcqxNDU3nmcuG58uRGq0vp07pE

**Comment by zsna** [comment]
![Snipaste_2022-05-19_20-23-41](https://user-images.githubusercontent.com/105859817/169293095-0210281a-050d-492b-84cc-5d86398ae48c.jpg)
![Snipaste_2022-05-19_20-23-52](https://user-images.githubusercontent.com/105859817/169293102-b89e6ec2-7e53-42b2-b36b-e94cdaf82f30.jpg)
How to solve the problem after filling in the key?

**Comment by nbzottel** [comment]
> @nbzottel https://mega.nz/file/wQl0XISA#khEE6DQ2TM6OvvGExgcqxNDU3nmcuG58uRGq0vp07pE

You absolute legend! 

I am still struggling to work out Spector via the tutorial; my results look a bit different than the correctly exported image you have sent me. But I will give it another go. ;)

**Comment by nbzottel** [comment]
> @nbzottel the texture are encrypted and no actual fix for that , the only thing to do is to extract the textures manually using Firefox/chrome ( [https://spector.babylonjs.com/](https://github.com/roro08/Sketchfab-Ripper/issues/url) ) extension called spector.js here is a quick tutorial : https://www.youtube.com/watch?v=klttU44JAfQ

I am missing the 3d+2d side-by-side viewport from the tutorial. Maybe this one is needed because I am not getting the uv-flattened textures, only screencaptures of the full 3d model.

I know that this can be disabled by the uploader (https://help.sketchfab.com/hc/en-us/articles/115004862686-Model-Inspector#viewport) but you managed to export the texture. Did you do something differently? Are you seeing a 3d model +2d texture side-by-side? 

(Sorry for all the questions but you have been a great help so far! :))

**Comment by moumouh206** [comment]
@nbzottel this model doesn't have 2D + 3D side by side viewport, it helps speedup the process, but it's not necessary for Spector to run

**Comment by nbzottel** [comment]
Sadly, I don't. So I have to be quick. ;) Thanks again for all your help!

**Comment by xIFIx** [comment]
> @nbzottelтекстуры зашифрованы, и никаких реальных исправлений для этого нет, единственное, что нужно сделать, это извлечь текстуры вручную с помощью расширения Firefox/chrome ( [https://spector.babylonjs.com/](https://github.com/roro08/Sketchfab-Ripper/issues/url) ), называемого spector.js , вот краткое руководство: [https ://www.youtube.com/watch?v=klttU44JAfQ](https://www.youtube.com/watch?v=klttU44JAfQ)

Can you throw off the link where you can find new keys in order for this program to work

**Comment by timurallanazarov** [comment]
> @nbzottel sfConfig=qklk1fVnWZo0mPrdNuQhoe2HvI2vRcxYw4K8qoyKTK1kg5Aq3J6CE2+6EEaJcwOR4GbaSnybqBmxe4NK5nwY2BZ5hgO3mnyCK27bWH9Upntb2P5H7Rg8UjTrCWNn8ZBX Only if you know some encryption, you can do it for your self
Where can I find new keys, can you tell?

**Comment by timurallanazarov** [comment]
> @nbzottel 
Where can I find new keys, can you tell?

**Comment by nbzottel** [comment]
No. @moumouh206 is probably clever enough to generate these keys because of his knowledge about encryption. I do not know anything about that and rely on him as well. Currently, there is no other way than asking nicely for a new, up-to-date decription key. :)

**Comment by moumouh206** [comment]
new key :
sfConfig=DXb9Nw3w+U9HBtwmZeTs0VU3LJ4qjqx8uB1ALqE2JF+I38b16SGRzC12O0gqzK1LwaL08PTP3qtbnxeUXV4jP/sRO2De4OLpD5uFmjm025Z6bzOM568SJjgaVAx2vRQ2

**Comment by ZooHairDev** [comment]
I have the same problem even though I tried all the keys

**Comment by jkattr** [comment]
I'm having the same problem and tried all the keys too

**Comment by moumouh206** [comment]
sfConfig=tZE+n6VM9ceEz60a4qQ0gkJhHqQnKuvisKJTLXwgFQNARFX2//O/cD7EWPlLoztfbapQjLWDOW9ZzsMC00hJtCZgtFdd9BhB1u+F59EY+/aNU9d2TMP9RdDEZF8uRt6c

**Comment by pan987-lxh** [comment]
I have the same problem even though I tried all the keys
